docs/flashbots-mev-boost/security.md

```diff
@@ -0,0 +1,7 @@
+# Security Audits
+
+- [20220620](https://github.com/flashbots/mev-boost/blob/main/docs/audit-20220620.md), by [lotusbumi](https://github.com/lotusbumi).
+
+### Bug Bounties
+
+- Coming soon!
```

Comment:
maybe add 

If you find a security vulnerability on this project or any other initiative related to Flashbots, please let us know sending an email to [security@flashbots.net](mailto:security@flashbots.net).

Reply:
added, although its duplicated under contributor section. Important enough to duplicate.